README.md

```diff
@@ -19,14 +19,18 @@ Design an API endpoint that provides auto-complete suggestions for large cities.
 - *You can use the language and technology of your choosing.* It's OK to try something new (tell us if you do), but feel free to use something you're comfortable with. We don't care if you use something we don't; the goal here is not to validate your knowledge of a particular technology.
 - End result should be deployed on a public Cloud (Heroku, AWS etc. all have free tiers you can use).
 
-## Advices
+## Advice
 
 - **Try to design and implement your solution as you would do for real production code**. Show us how you create clean, maintainable code that does awesome stuff. Build something that we'd be happy to contribute to. This is not a programming contest where dirty hacks win the game.
-- Feel free to add more features! Really, we're curious about what you can think of. We'd expect the same if you worked with us.
-- Documentation and maintainability is a plus.
-- Don't you forget those unit tests.
+- Documentation and maintainability are a plus, and don't you forget those unit tests.
 - We don’t want to know if you can do exactly as asked (or everybody would have the same result). We want to know what **you** bring to the table when working on a project, what is your secret sauce. More features? Best solution? Thinking outside the box?
 
+## Can I use a database?
+
+If you wish, it's OK to use external systems such as a database, an Elastic index, etc. in your solution. But this is certainly not required to complete the basic requirements of the challenge. Keep in mind that **our goal here is to see some code of yours**; if you only implement a thin API on top of a DB we won't have much to look at.
+
```

Comment:
> only implement a thin API on top of a DB

hihi this is pretty much what I did 😬

Reply:
Yeah we're trying to avoid future Davids here.